### PR TITLE
a couple of fixes I ran into

### PR DESCRIPTION
**Pull request by SwissalpS**
- two asterisks in readme
- syntax fix in dateTimeStamps
- fixed dragdropselected from changes to xl
- dirty patch to indentguide so it doesn't crash xl

### CODE REVIEW

**Review by Guldoman — COMMENTED** [review]
I can't test it now, but what is the purpose of the changes to `dragdropselected`?

**Comment by SwissalpS** [comment]
> I can't test it now, but what is the purpose of the changes to `dragdropselected`?

enables dragging selected portions, moving it to where dropped.
Before the change, only copy (with ctrl-key) was possible, moving didn't work.

File: plugins/indentguide.lua [hunk 1]
```diff
@@ -39,12 +39,13 @@ function DocView:draw_line_text(idx, x, y)
   local h = self:get_line_height()
   local sspaces = ""
   local font = self:get_font()
-  local ss = font:subpixel_scale()
+  local ss = 1 --font:subpixel_scale()
   for _ = 0, spaces - 1, config.indent_size do
     local color = style.guide or style.selection
-    local sw = font:get_width_subpixel(sspaces) / ss
+    local sw = font:get_width(sspaces) / ss --font:get_width_subpixel(sspaces) / ss
```
[the remark this answers is not in the document]
Comment (SwissalpS) [reply plugins/indentguide.lua:45]: how is it correct? I didn't know how to fix it other than making it not crash anymore.

[the remark this answers is not in the document]
Comment (Guldoman) [reply plugins/indentguide.lua:45]: This change will be done by #1 when 2.1 releases.
If you're using `master` you'll have to keep this change locally.

**Comment by Guldoman** [comment]
> enables dragging selected portions, moving it to where dropped. Before the change, only copy (with ctrl-key) was possible, moving didn't work.

The current version works correctly on 2.0.4. On `master` it will need some small fixes.

**Comment by SwissalpS** [comment]
closing because stale and hugely out of date

**Pull request closed without merging**